pkg/test/framework/components/istioctl/kube.go

```diff
@@ -65,7 +65,9 @@ func (c *kubeComponent) Invoke(args []string) (string, error) {
 func (c *kubeComponent) InvokeOrFail(t *testing.T, args []string) string {
 	output, err := c.Invoke(args)
 	if err != nil {
-		t.Fatalf("Unwanted exception for 'istioctl %s': %v", strings.Join(args, " "), err)
+		t.Logf("Unwanted exception for 'istioctl %s': %v", strings.Join(args, " "), err)
```

Comment:
These should probably be logged as errors. Maybe use t.Errorf?

Reply:
I used Logf because the doc comment for Fatalf says "Fatalf is equivalent to Logf followed by FailNow."

Errorf has a slightly different behavior, equivalent to Logf + Fail. So Errorf would mark the test as failed, but not halt it. While I think there's a case to be made for that behavior, the rest of the `DoFooOrFail` test helpers I've looked at use a fatal variant to fail immediately, and I think it makes sense to stay consistent with them.